docs/components/identity/authorization.md

```diff
@@ -0,0 +1,51 @@
+---
+id: authorization
+title: Authorization
+description: "Learn how to manage authorizations in your Orchestration cluster."
+---
+
+Identity provides unified, cluster-level identity management and authorizations. Authorizations are managed through permissions that can be assigned to [users](user.md), group, role and mapping.
+
+## Authorization overview
+
+Authorization is a mechanism that controls access to resources and actions based on permissions assigned to an entity.
+
+Camunda 8 cluster uses resource-based authorization control, which means that permissions are granted to entities (like users) to perform specific actions on specified resources. For example, permissions can be set for actions such as reading, updating, creating, or deleting resources like process instances.
+
+Authorizations can be enabled or disabled for each Camunda 8 cluster via [configuration settings](/self-managed/identity/orchestration-identity/installation.md#enable-api-authentication-and-authorizations). By default, authorizations are disabled. When they are enabled, a user is not granted access to anything by default and must be explicitly granted permission to access applications or perform actions on resources.
+
+## Authorization concepts
```

Comment:
❓ Are we describing this in a general sense or a C8 sense? I think here I would opt (its its about C8) to be more descriptive with an example, in C8 an authorization has 3 core parts, the first is an owner, the second are resources and the third is the permissions allowed on those resources. This is how I create the authorization in the UI.

In the future I think we can be quite visual here and create a graphic of an owner (person) a BPMN (resource id) and permission. As a text example:

Owner: {Type: User, ID: foo@Bar} has permissions: ["Read process definition", "Update process definition"] for resources: ["my-bpmn-process", "my-other-bpmn"]

This can then be extended to show groups and how we handle inheritance

Reply:
I can add an owner section and explain what does it mean in C8